R/track_rust_source.R

```diff
@@ -48,11 +49,11 @@ pretty_rel_path <- function(path, search_from = ".") {
   # return `path` unchanged (for simplicity).
   if (
     !nzchar(package_root) ||
-      !stringi::stri_detect_fixed(
+      !all(stringi::stri_detect_fixed(
```

Comment:
This is not exactly the expected behavior. I'd suggest to keep this function as is, but rename it. And then introduce the new `pretty_rel_path()` as 
```R
pretty_rel_path <- function(path, search_from = ".") {
   purrr::map_chr(path, pretty_rel_path_impl, search_from = search_from)
}
```
to add proper vectorization support.